src/sass/components/_contacts.scss

```diff
@@ -0,0 +1,127 @@
+@import '../variables';
+
+// @media (min-width: 320px) {
+    * {
+        margin: 0 0;
+    }
+
+    .contacts {
+        // margin: 86px 0px 113px 0px;
+        background: linear-gradient(0deg, rgba(255, 245, 246, 1) 97%, rgba(255,255,255,1) 97%);
+        // without margins: full 1473px; pink: 1433px; white: 40px
+        
+        &__wrapper {            
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            gap: 20px;
+        }
+
+        &__list {
+            padding: 43px 21px 43px 20px;
+            max-width: 220px;
+            background: var(--white);
+            box-shadow: var(--shadow);
+            border-radius: 24px;
+        }
+
+        &__item {
+            border-bottom: 1px solid var(--footerborder);
+
+            &--margin-thicc {
+                margin-bottom: 25px;
+            }
+
+            &--margin-slender {
+                margin-bottom: 10px;
+            }
+        }
+
+        &__shop-type {
+            margin-bottom: 25px;
+            height: 100%;
+            border-radius: 4px;
+            font-weight: 500;
+            font-size: 12px;
+            line-height: 186%;
+            letter-spacing: 0.04em;
+            color: var(--white);
+            text-align: center;
```

Comment:
pierwszy z buttonow nie rendreuje sie tak jak na makiecie, ikonka prechodzi na druga linijke, przynajmniej tak to u mnie wyglada. moze zamiast text-align:center sprobowac flex -> justify-content: space around, albo cos podobnego?

Reply:
hmmm, u mnie to wygląda prawidłowo, ciekawe

Reply:
<img width="639" alt="image" src="https://user-images.githubusercontent.com/117107541/215294894-6d4d55e9-2944-4705-aa75-c630abd2bef7.png">

Reply:
ciekawy jestem od czego to może zależeć, ta różnica
![image](https://user-images.githubusercontent.com/117771397/215295079-099c8031-c031-4c54-ae0d-0f03996dd5d9.png)

Reply:
usunąłem padding w button, jakby ktoś mógł zobaczyć czy teraz wektor prawidłowo się wyświetla (w jednej linii)

Reply:
ja tez korzystam z chrome, ale teraz wyglada juz jak trzeba
<img width="566" alt="image" src="https://user-images.githubusercontent.com/117107541/215295379-f7c5472c-7019-4bd7-9418-be53af9e1d9a.png">